subsystem/kubeapi_test.go

```diff
@@ -881,73 +872,74 @@ var _ = Describe("[kube-api]cluster installation", func() {
 		}, "2m", "2s").Should(Equal(0))
 	})
 
-	It("deploy None platform CD with ACI and agents - wait for ready, delete CD and verify ACI and agents deletion", func() {
```

Comment:
Why change an existing test and not add a new one?

Reply:
+1